Macros/e3 Includes/e3_Classes_Bard.inc

```diff
@@ -202,21 +220,24 @@ sub check_bard_mez
     /declare i int local
     /declare s int local
     /declare mezTarg int local
-    /if (!${SpawnCount[playerstate 4 npc targetable radius ${AE_Radius}]}) /return
+    /if (!${SpawnCount[playerstate 4 npc targetable radius ${AE_Radius}]}) {
+      /varset returnTwist TRUE
+      /return
+    }
     /for i 1 to ${SpawnCount[playerstate 4 npc targetable radius ${AE_Radius}]}
       /varset mezTarg ${NearestSpawn[${i},playerstate 4 npc targetable radius ${AE_Radius}].ID}
       /if (${Spawn[id ${mezTarget}].CleanName.Find[' pet]}) /next i
       /call TrueTarget ${mezTarg}
       /delay 1s ${Target.BuffsPopulated}
-      /if (${Select[${AssistTarget},${mezTarg}]}==0 && (!${Bool[${Target.Mezzed}]} || ${Target.Mezzed.Duration.TotalSeconds} < 5)) {
+      /if (${Select[${AssistTarget},${mezTarg}]}==0 && (!${Bool[${Target.Mezzed}]} || ${Target.Mezzed.Duration.TotalSeconds} < 7)) {
         /for s 1 to ${mezArray2D.Size[1]}
           /if (${check_Distance[${mezTarg},${mezArray2D[${s},${iMyRange}]}]}) {
             /if (${mezArray2D[${s},${iCastType}].Equal[Spell]} && !${Bool[${Me.Gem[${mezArray2D[${s},${iCastName}]}]}]}) /call memorize_spell "${mezArray2D[${s},${iCastName}]}" ${mezArray2D[${s},${iSpellGem}]}
               /docommand ${ChatToggle} << Mezzing ${mezTarg} with ${mezArray2D[${s},${iCastName}]} >>
               /if (${Twist.Twisting}) /call pauseTwist
               |:retryMez
               /twist once ${mezArray2D[${s},${iSpellGem}]}
-              /delay 3s !${Me.Casting.ID}
+              /delay 3s 
```

Comment:
The !${Me.Casting.ID} should cause it to break out early if it finishes casting. Is that a problem for some reason?

Reply:
Very odd